apps/condo/domains/notification/constants/constants.js

```diff
@@ -340,23 +345,126 @@ const MESSAGE_META = {
             url: { defaultValue: '', required: true },
         },
     },
-    [CUSTOM_CONTENT_MESSAGE_TYPE]: {
+    [CUSTOM_CONTENT_MESSAGE_PUSH_TYPE]: {
         dv: { required: true },
         title: { required: false },
-        subject: { required: false },
         body: { required: true },
         data: {
             userId: { required: false },
+            remoteClient: { required: false },
             url: { defaultValue: '', required: false },
             messageBatchId: { required: false },
         },
     },
+    [CUSTOM_CONTENT_MESSAGE_EMAIL_TYPE]: {
+        dv: { required: true },
+        title: { required: false },
+        subject: { required: false },
+        body: { required: true },
+        data: {
+            email: { required: false },
+            messageBatchId: { required: false },
+        },
+    },
+    [CUSTOM_CONTENT_MESSAGE_SMS_TYPE]: {
+        dv: { required: true },
+        title: { required: false },
+        body: { required: true },
+        data: {
+            phone: { required: false },
+            messageBatchId: { required: false },
+        },
+    },
+}
+
+
+
+const MESSAGE_DELIVERY_STRATEGY_AT_LEAST_ONE_TRANSPORT = 'atLeastOneTransport'
+const MESSAGE_DELIVERY_STRATEGY_ALL_TRANSPORTS = 'allTransports'
+const MESSAGE_DELIVERY_DEFAULT_PRIORITY = 'default'
+const MESSAGE_DELIVERY_SLOW_PRIORITY = 'slow'
```

Comment:
is the priority out of the scope of this PR?

Reply:
found the way to implement it.